wallet-new/server/Cardano/Wallet/API/V1/Swagger.hs

```diff
@@ -255,38 +257,63 @@ Software Version   | Git Revision
+send a client CA certificate that was used when launching the node and identifies the client as
+being permitted to invoke the server API.
 
 Creating a New Wallet
 ---------------------
 
-You can create your first wallet using the `POST /api/v1/wallets` endpoint as follow:
+You can create your first wallet using the [`POST /api/v1/wallets`](#tag/Wallets%2Fpaths%2F~1api~1v1~1wallets%2Fpost) endpoint as follow:
 
 ```
-curl -X POST https://localhost:8090/api/v1/wallets                     \
-     -H "Content-Type: application/json; charset=utf-8"                \
-     -H "Accept: application/json; charset=utf-8"                      \
-     --cacert ./scripts/tls-files/ca.crt                               \
-     -d '{                                                             \
-  "operation": "create",                                               \
-  "backupPhrase": ["squirrel", "material", "silly", "twice", "direct", \
-    "slush", "pistol", "razor", "become", "junk", "kingdom", "flee"],  \
-  "assuranceLevel": "normal",                                          \
-  "name": "MyFirstWallet"                                              \
+curl -X POST https://localhost:8090/api/v1/wallets \
+  -H "Accept: application/json; charset=utf-8" \
+  -H "Content-Type: application/json; charset=utf-8" \
+  --cacert ./scripts/tls-files/ca.crt \
+  -d '{
+  "operation": "create",
+  "backupPhrase": ["squirrel", "material", "silly", "twice", "direct",
+    "slush", "pistol", "razor", "become", "junk", "kingdom", "flee"],
+  "assuranceLevel": "normal",
+  "name": "MyFirstWallet",
+  "spendingPassword": "5416b2988745725998907addf4613c9b0764f04959030e1b81c603b920a115d0"
 }'
 ```
 
-> **Warning**: Those 12 mnemonic words given for the backup phrase act as an example. **Do not** use them on a production system. See the section below about mnemonic codes for more information.
+> **Warning**: Those 12 mnemonic words given for the backup phrase act as an example. **Do
+> not** use them on a production system. See the section below about mnemonic codes for more
+> information.
 
-As a response, the API provides you with a unique wallet `id` to be used in subsequent requests. Make sure to store it / write it down. Note that every API response is [jsend-compliant](https://labs.omniti.com/labs/jsend); Cardano also augments responses with meta-data specific to pagination. More details in the section below about *Pagination*.
+The `spendingPassword` is optional but highly recommended. It is an original UTF-8
+string of 32 characters, encoded in base 16, yielding to an hexadecimal sequence of 64 bytes.
+This passphrase is required for sensitive operations on the wallet and adds
+therefore an extra security layer to it.
+
+To generate a valid `spendingPassword`, please follow the following steps:
+
+    - Pick a strong passphrase (long, from a big set of characters), randomly
+      generated passphrase are best for humans are a poor source of randomness.
```

Comment:
Clarity: Rather than qualifying what is meant by `strong`, we can just state it directly:

> Pick a long (how many characters/words?) passphrase using a wide variety of characters (uppercase, lower case, whitespace, punctuation, etc).

I'd probably edit the sentence to say: "Using a computer to randomly generate a passphrase is best, as humans aren't a good source of randomness," though the only grammar error is "randomly generated passphrase*s* are ..." or "*a* randomly generated passphrase *is* best..."